src/index.ts

```diff
@@ -332,7 +222,7 @@ function activateGalyleo(
     fileTypes: ['Galyleo'],
     defaultRendered: ['Galyleo'],
     defaultFor: ['Galyleo'],
-    modelName: 'galyleo',
```

Comment:
...and you can simply tell the widget factory it needs to use the default text model factory.